http-filter-example/http_filter_v2_integration_test.cc

```diff
@@ -0,0 +1,82 @@
+#include "common/grpc/codec.h"
```

Comment:
Nit: you can probably keep the old test name `http_filter_integration_test`. Either that, or update docs at https://github.com/envoyproxy/envoy-filter-example/blob/master/http-filter-example/README.md.

Reply:
@htuch given that v2 is going to be default choice, I changed the test name back to http_filter_integration_test rather than updating docs.